net/wfb-ng/files/wfb-ng.init

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=10
+
+USE_PROCD=1
+NAME=wfb-ng
+
+SYSUPGRADE_CONF="/etc/sysupgrade.conf"
+WFB_INIT="/usr/sbin/wfb-ng.sh"
+
+if [ -f $SYSUPGRADE_CONF ] && [ -x $WFB_INIT ] && ! grep -q $WFB_INIT $SYSUPGRADE_CONF
+then
+   echo $WFB_INIT >> $SYSUPGRADE_CONF
+fi
```

Comment:
I am not sure, what you would like to achieve with this one? You want to persist init file through sysupgrade? Do you know how the sysupgrade works?

Reply:
`/usr/sbin/wfb-ng.sh` is a custom init script, generated by user. I want to persist it during sysupgrade

Reply:
Definitely, this should not be here. Usually there is section conffiles, but in that case, the file is mentioned in the Makefile.

Reply:
How to add `/usr/sbin/wfb-ng.sh` to /etc/sysupgrade.conf in the Makefile? I've tried to search this in other openwrt packages, but found only the way above in `net/p910nd/files/p910nd.hotplug`

Reply:
define a conffiles recipe.

FYI:
https://github.com/openwrt/packages/blob/99c28c28d2bfa976a52d66fae8630fb2823bf583/net/alist/Makefile#L47-L50

Reply:
@1715173329 fixed. Thanks!

Reply:
The question is if this is going to work, though. Needs to be tested for files, which are not included/presented in the Makefile.

Reply:
@BKPepe Alredy done. Package was built successbully

Reply:
Also i've checked it on real hardware (TP-Linke CPE510v3).